manifests/deployment.yaml

```diff
@@ -1,22 +1,22 @@
-apiVersion: apps/v1
+apiVersion: extensions/v1beta1
```

Comment:
I think there is no need to update here.

Reply:
Yes, I just make it same as pytorch. I will keep it if no issue is introduced. :)